shell/platform/android/io/flutter/embedding/android/FlutterImageView.java

```diff
@@ -99,6 +109,20 @@ private static void logW(String format, Object... args) {
     Log.w(TAG, String.format(Locale.US, format, args));
   }
 
+  private void setOnImageAvailableListener() {
+    if (imageReader != null) {
+      imageReader.setOnImageAvailableListener(this, null);
+    }
+  }
+
+  /** OnImageAvailableListener callback. */
+  @Override
+  public void onImageAvailable(ImageReader reader) {
```

Comment:
This should not be part of the public API of FlutterImageView

Reply:
removed